src/bp/core/services/nlu/intent-service.ts

```diff
@@ -18,7 +19,45 @@ export class IntentService {
 
   public async getIntents(botId: string): Promise<sdk.NLU.IntentDefinition[]> {
     const intentNames = await this.ghostService.forBot(botId).directoryListing(INTENTS_DIR, '*.json')
-    return Promise.mapSeries(intentNames, n => this.getIntent(botId, n))
+    const intentsFromFiles = await Promise.mapSeries(intentNames, n => this.getIntent(botId, n))
```

Comment:
do you want to keep this to be backward compat ? //  TODO deprecate this ?

Reply:
Add TODO to move the built-in "Yes", "No", "Cancel" in `/library/built-in.intents.json`